### DISCUSSION TITLE
Puffer

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#268] [Show and tell]
Puffer is a free Stanford University research project that web streams California Bay Area stations for NBC, CBS, ABC, PBS, Fox, and CW networks.  Any chance the streams could be found and added to IPTV-org? Doing so might benefit us and the Stanford project. According to their FAQ;  "The more diverse the Internet connections that the study participants use, the better the system will be able to learn, and the more robust the resulting computer-generated algorithms."

Sign up (does not require an e-mail address) and log in to watch:
https://puffer.stanford.edu/
Puffer's source code:
https://github.com/StanfordSNR/puffer
FAQ:
https://puffer.stanford.edu/faq/

![image](https://user-images.githubusercontent.com/88639906/146447592-02b9567e-a067-4aa8-94e7-839a03360150.png)

**Comment by MapGuy11** [comment]
> Puffer is a free Stanford University research project that web streams California Bay Area stations for NBC, CBS, ABC, PBS, Fox, and CW networks.  Any chance the streams could be found and added to IPTV-org? Doing so might benefit us and the Stanford project. According to their FAQ;  "The more diverse the Internet connections that the study participants use, the better the system will be able to learn, and the more robust the resulting computer-generated algorithms."
> 
> Sign up (does not require an e-mail address) and log in to watch:
> https://puffer.stanford.edu/
> Puffer's source code:
> https://github.com/StanfordSNR/puffer
> FAQ:
> https://puffer.stanford.edu/faq/
> 
> ![image](https://user-images.githubusercontent.com/88639906/146447592-02b9567e-a067-4aa8-94e7-839a03360150.png)
> 
> 
> 

No they use web rtc.

**Reply by BellezaEmporium** [reply]
Streamlink seems to be supporting *certain* websocket streams. I don't know how would it work in this case.
